presto-product-tests/README.md

```diff
@@ -225,10 +225,11 @@ setup outlined below:
     ./mvnw install -DskipTests
     ```
 
-2. Start Hadoop in pseudo-distributed mode in a Docker container:
+2. Start Presto dependant services as Docker containers:
 
     ```
-    presto-product-tests/conf/docker/singlenode/compose.sh up -d hadoop-master
+    presto-product-tests/conf/docker/singlenode/compose.sh up -d 
```

Comment:
I would rather specify all the required services explicitly here. I don't think we are going to add new ones to often. And anyway, if we added yet another one, documentation update would be required.